### DISCUSSION TITLE
improve the search

### DISCUSSION THREAD

**Discussion by sunsunmonkey** [pmndrs/zustand#2342] [Ideas] [answered]
I find this api in source code. But I don't found it in the documents.

**Comment by dai-shi** [comment]
Check this out:  https://github.com/pmndrs/zustand/blob/8c1a1f0327db3790e19a8bccd1fd47042683f4f9/docs/guides/prevent-rerenders-with-use-shallow.md?plain=1#L2

**Reply by sunsunmonkey** [reply]
![image](https://github.com/pmndrs/zustand/assets/116412388/9263c594-8f2b-4721-a72e-195816a2d937)
sorry,but it can't find by searching🙃

**Comment by dai-shi** [comment]
> improve the search

hmm, it looks like the index isn't updated.

I'm not very familiar with how it's implemented, but the code is here: https://github.com/pmndrs/website/tree/docs

**Reply by dbritto-dev** [reply]
@dai-shi I believe we need to fix something here -> https://github.com/pmndrs/website/blob/docs/src/utils/docs.ts#L49

**Reply by dai-shi** [reply]
Did you already find how to fix?
Yeah, PR would be appreciated.

**Reply by dbritto-dev** [reply]
@dai-shi I've been playing with that but I could have a PR tomorrow

**Reply by dbritto-dev** [reply, marked as answer]
@dai-shi here you go -> https://github.com/pmndrs/website/pull/273

cc: @sunsunmonkey
